packages/myst-cli/src/build/utils/collectExportOptions.ts

```diff
@@ -312,11 +313,16 @@ export function resolveOutput(
     exp.zip = false;
   }
   if (!path.extname(output)) {
-    const basename = getDefaultExportFilename(
-      session,
-      singleArticleWithFile(exp.articles)?.file ?? sourceFile,
-      projectPath,
-    );
+    let basename: string;
+    if (exp.format === ExportFormats.cff) {
+      basename = 'CITATION';
```

Comment:
The specific filename `CITATION.cff` is part of the spec, so this format does not follow the normal export naming pattern.